### DISCUSSION TITLE
Hybrid schema files

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#602] [Q&A]
Is there any reason that some payers are posting hybrid files?  These files combine features of the table-of-contents and features of the in-network schemas.  An example is [2022-09-01_BSW_Plus_HMO_INN.json](https://rightcare.swhp.org/Portals/5/In-Network/2022-09-01_BSW_Plus_HMO_INN.json) on this page:

https://www.bswhealthplan.com/Pages/TransparencyInCoverage.aspx

It has both a report_plans section and then an in_network section rather than an in_network_files, so it isn't exactly following the table-of-contents schema or the in-network schema.

**Comment by dgolden** [comment]
On a positive note, at least they are pushing data out. 
It can be really hard to get creative people, including software developers. to "build to spec."
Any of the larger payers doing this?
